src/egui_skia.rs

```diff
@@ -87,25 +86,27 @@ impl EguiSkia {
         &mut self,
         input: egui::RawInput,
         run_ui: impl FnMut(&Context),
-    ) -> (Duration, egui::PlatformOutput) {
+    ) -> egui::PlatformOutput {
         let egui::FullOutput {
             platform_output,
             textures_delta,
             shapes,
-            repaint_after,
```

Comment:
Did repaint_after get removed form the ouput or why did you remove it?

Reply:
Yes:
``` 
error[E0026]: struct `FullOutput` does not have a field named `repaint_after`
  --> /home/oscgu95/.cargo/registry/src/index.crates.io-6f17d22bba15001f/egui_skia-0.4.0/src/egui_skia.rs:95:13
   |
95 |             repaint_after,
   |             ^^^^^^^^^^^^^ struct `FullOutput` does not have this field

```

Reply:
Ah, interesting 👍